# RNA-seq Analysis of Kidney Renal Papillary Cell Carcinoma
### _Bioinformatics Resources Project_

---
## Frederick , FAC, Abi Chahine
> M.Sc. in Quantitative & Computational Biology, University of Trento, Italy
## Hala, HA, Alshaar
> M.Sc. in Quantitative & Computational Biology, University of Trento, Italy 
## Anastasia, AB, Bertova
> M.Sc. in Quantitative & Computational Biology, University of Trento, Italy

---

## Pipeline  
  
This project focuses on the analysis of **RNA-seq** count data derived from KIRP samples obtained from **The Cancer Genome Atlas** (TCGA). The goal is to explore the molecular differences between **tumor and normal tissues** through a comprehensive bioinformatics pipeline. Key steps include **data preprocessing, filtering for protein-coding genes** using the *biomaRt package*, and **identifying DEGs** via the *edgeR package*. **Visualization** techniques such as *volcano plots and heatmaps* are used to interpret the differential expression results. **Gene set enrichment analysis** (GSEA) is performed using the *fgsea package* to uncover significantly altered pathways in cancer. In addition, we **investigate transcription factors** (TFs) with enriched binding motifs in the promoters of regulated genes and **examine protein-protein interaction** (PPI) networks using *STRING and igraph*. This multi-layered approach provides insights into the transcriptional landscape of KIRP and highlights potential biomarkers or therapeutic targets for further research.  

## Report

ðŸ“„ [Click here to view the full report](https://frederick-abichahine.github.io/KIRP-RNAseq-Analysis/KIRP_RNAseq_analysis.html)
